### PR TITLE
add and fetch users

### PR DESCRIPTION
**Pull request by goofballLogic**

### CODE REVIEW

**Review by slawomircieslinski — CHANGES_REQUESTED** [review]
I'm getting  415 unsupported media type for https://localhost:5001/users/1

**Comment by goofballLogic** [comment]
> I'm getting 415 unsupported media type for https://localhost:5001/users/1

thanks - should be fixed now

**Review by slawomircieslinski — APPROVED** [review]

File: Hubl.LoginAPI/Hubl.LoginAPI/UsersController.cs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        public IActionResult Index([FromRoute] int id, [FromBody] User user)
+        {            
+            using var cnn = new SqlConnection("Server=localhost;Database=hublsoft-login;Trusted_Connection=true");
+            cnn.Open();
+            using var cmd = cnn.CreateCommand();
+            if (Request.Method == "GET")
+            {                
+                return GetUser(id, cmd);
+            }
+            else if (Request.Method == "PUT")
```
[the remark this answers is not in the document]
Comment (goofballLogic) [reply Hubl.LoginAPI/Hubl.LoginAPI/UsersController.cs]: thanks - POST would be better here.

File: Hubl.LoginAPI/Hubl.LoginAPI/UsersController.cs [hunk 2]
```diff
@@ -57,5 +46,21 @@ private IActionResult GetUser(int id, SqlCommand cmd)
             }
             return BadRequest();
         }
+
+        private IActionResult Process(int id, User user)
+        {
+            using var cnn = new SqlConnection("Server=localhost;Database=hublsoft-login;Trusted_Connection=true");
+            cnn.Open();
+            using var cmd = cnn.CreateCommand();
+            if (Request.Method == "GET")
```
Comment (slawomircieslinski) [inline Hubl.LoginAPI/Hubl.LoginAPI/UsersController.cs:55]: Why Request.Method decide which sp should be called, what if we have to call GetUserAudit or DeleteUser

Comment (goofballLogic) [reply Hubl.LoginAPI/Hubl.LoginAPI/UsersController.cs:55]: What would a better solution be for those?

Comment (slawomircieslinski) [reply Hubl.LoginAPI/Hubl.LoginAPI/UsersController.cs:55]: Maybe Routing. We can match URLs of incoming requests and map them to actions?

Comment (goofballLogic) [reply Hubl.LoginAPI/Hubl.LoginAPI/UsersController.cs:55]: ok let me put another version in and see what you think

Comment (goofballLogic) [reply Hubl.LoginAPI/Hubl.LoginAPI/UsersController.cs:67]: Ok I have added in the Delete user method (although not the GetUserAudit one yet). I also have three methods (lines 14, 17, 20) which correspond to each route. Is that what you were thinking?

Comment (slawomircieslinski) [reply Hubl.LoginAPI/Hubl.LoginAPI/UsersController.cs:67]: Something like this, I even added GetUserAudit
https://github.com/slawomircieslinski/hublsoft-login/compare/login-API...Amethyst#diff-849319779895a0ba7c576a67c75f8a97

**Pull request merged**